### PR TITLE
Guide About Business Rules Client Lib

### PR DESCRIPTION
**Pull request by Johannes-Schneider**
## What Has Changed?

This PR adds a guide on how to use the pre-generated Business Rules client library.

## Manual Checks?

- [x] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/4HANA)
- [x] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [x] You formatted all changed files with prettier (`npm run prettier`)
- [x] You tested if the documentation still builds (`npm run build`)
- [x] You verified all new and changed links still work (changing the `id` or name of a file can break links)
- [x] You have updated the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js) if you add documentation on a new feature or otherwise required.

### CODE REVIEW

File: docs/java/features/rest/clients/btp-business-rules-rest-api.mdx [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+Current version is: <MvnBadge />
+:::
+
+```xml title="pom.xml"
+<dependencyManagement>
+  <dependencies>
+    <dependency>
+      <groupId>com.sap.cloud.sdk</groupId>
+      <artifactId>sdk-bom</artifactId>
+      <!-- Business Rules API is supported in ver 3.45.0 of the SDK and above. Please, always use the latest version -->
```
Comment (Johannes-Schneider) [inline docs/java/features/rest/clients/btp-business-rules-rest-api.mdx:118]: Wait for this version to hit Maven Central before merging this PR.

File: docs/java/features/rest/clients/btp-business-rules-rest-api.mdx [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+      SET_LOGGING_LEVEL: '{ROOT: INFO, com.sap.cloud.sdk: INFO}'
+      JBP_CONFIG_SAPJVM_MEMORY_SIZES: metaspace:128m..
+    services:
+      - my-destination
+      - my-business-rules
+    routes:
+      - route: <omitted-on-purpose>
+```
+
+This ensures that the `VCAP_SERVICES` environment variable of the CF application gets enhanced with an additional entry containing the connection details of the newly bound workflow service.
```
Comment (artemkovalyov) [inline docs/java/features/rest/clients/btp-business-rules-rest-api.mdx:95]:
```suggestion
This ensures that the `VCAP_SERVICES` environment variable of the CF application gets enhanced with an additional entry containing the connection details of the newly bound SAP BTP Business Rules service.
```

File: docs/java/features/rest/clients/scp-workflow-rest-api.mdx [hunk 3]
```diff
@@ -265,7 +264,7 @@ The steps below can be continued in this example from the [Bind your App to Serv
 
 ### Take Note of API endpoint and OAuth Credentials
 
-Once the app is bound to the workflow service (here in our example) and app deployment has finished, go to your CF space and navigate to `Services\Service Instances`.
+Once the app is bound to the workflow service (here in our example) and app deployment has finished, go to your CF space and navigate to `Services\Instances and Subscriptions`.
```
Comment (artemkovalyov) [inline docs/java/features/rest/clients/scp-workflow-rest-api.mdx:267]:
```suggestion
Once the app is bound to the SAP BTP Business Rules service (here in our example) and app deployment has finished, go to your CF space and navigate to `Services\Instances and Subscriptions`.
```

File: docs/java/features/rest/clients/scp-workflow-rest-api.mdx [hunk 4]
```diff
@@ -265,7 +264,7 @@ The steps below can be continued in this example from the [Bind your App to Serv
...
 You should see the service instance you created along with the information that is bound to your application.
 
 Click on the service instance name, for instance `my-workflow`, in the upcoming screen you should see the headline `Service Instance: my-workflow - Referencing Apps`.
```
Comment (artemkovalyov) [inline docs/java/features/rest/clients/scp-workflow-rest-api.mdx:270]:
```suggestion
Click on the service instance name, in our case `my-business-rules`, in the upcoming screen you should see the headline `Service Instance: my-business-rules - Referencing Apps`.
```

**Review by artemkovalyov — CHANGES_REQUESTED** [review]
I added some suggestions. It's clear the docs are copied from Workflow and in many cases, it still seems to require an update. Or do I miss smth? Check my initial notes.

File: docs/java/features/rest/clients/scp-workflow-rest-api.mdx (hunk 3 above)
Comment (Johannes-Schneider) [reply docs/java/features/rest/clients/scp-workflow-rest-api.mdx:267]: This is in the `scp-workflow-rest-api` documentation, so here the "workflow service" is actually correct.

File: docs/java/features/rest/clients/scp-workflow-rest-api.mdx (hunk 4 above)
Comment (Johannes-Schneider) [reply docs/java/features/rest/clients/scp-workflow-rest-api.mdx:270]: Same as above.

File: docs/java/features/rest/clients/btp-business-rules-rest-api.mdx [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+The [SAP Business Rules service](https://help.sap.com/viewer/product/BUSINESS_RULES/Cloud/en-US) for Cloud Foundry helps customers to translate complex business logic into a natural language.
+It consists of multiple APIs that can be used for modelling, authoring, and simulating business rules.
+
+To benefit from features offered by the REST APIs of the SAP BTP Business Rules service you can leverage the type-safe client library provided by the **SAP Cloud SDK** and discover it via your IDE.
+
+### Prerequisites
+
+Add the latest version of the [SAP Cloud SDK](https://search.maven.org/artifact/com.sap.cloud.sdk/sdk-bom) to your Java application dependencies or [generate a new one from archetypes that we provide](../../../getting-started).
+
+After you created an SDK-based Java app, you can invoke the SAP BTP Business Rules REST API in your business logic.
```
Comment (artemkovalyov) [inline docs/java/features/rest/clients/btp-business-rules-rest-api.mdx:38]:
```suggestion
After you created an SAP Cloud SDKSDK-based Java app, you can invoke the SAP BTP Business Rules REST API in your business logic.
```

**Review by artemkovalyov — APPROVED** [review]
LGTM

File: docs/java/features/rest/clients/btp-business-rules-rest-api.mdx (hunk 5 above)
Comment (artemkovalyov) [inline docs/java/features/rest/clients/btp-business-rules-rest-api.mdx:38]:
```suggestion
After you created an SAP Cloud SDK-based Java app, you can invoke the SAP BTP Business Rules REST API in your business logic.
```

**Pull request merged**